halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/SpinnakerMonitoringDaemonService.java

```diff
@@ -75,6 +75,10 @@ private static String monitoringProfileName() {
     return "spinnaker-monitoring.yml";
   }
 
+  private static String monitoringLocalProfileName() {
+    return "spinnaker-monitoring.yml";
```

Comment:
Shouldn't this be `spinnaker-monitoring-local.yml`?

Reply:
good catch